source/utilities.tex

```diff
@@ -21135,3 +21135,685 @@
 \ensures
 \tcode{strcmp(what(), what_arg) == 0}.
 \end{itemdescr}
+
+\rSec1[stacktrace]{Stacktrace}
+
+\rSec2[stacktrace.general]{General}
+
+\pnum
+Subclause \ref{stacktrace} describes components
+that C++ programs may use to store
+the stacktrace of the current thread of execution and
+query information about the stored stacktrace at runtime.
+
+\rSec2[stacktrace.def]{Stacktrace definition}
```

Comment:
Won't ISO complain that these should go into [intro.defs] because it has definition in its name? Maybe they should be subtly introduced in the text, like all other definitions that don't go into [intro.defs].

Reply:
Removed the heading; it's now under "General".